config/config.go

```diff
@@ -108,6 +110,21 @@ type Configuration struct {
 	PriceFloors PriceFloors `mapstructure:"price_floors"`
 }
 
+type GoRuntime struct {
+	// SoftMemoryLimit is the soft limit for memory usage in bytes. The garbage collector will run
+	// more often if the memory usage is above this limit and less often when memory usage is far
+	// below this limit. SoftMemoryLimit in conjunction with GarbageCollectorTriggerPercent can be
+	// used to tune Garbage Collector performance.
+	// Please read the Go GC Guide for more info: https://tip.golang.org/doc/gc-guide
+	SoftMemoryLimit int `mapstructure:"soft_memory_limit"`
+	// GarbageCollectorTriggerPercent is the percentage of heap growth that triggers the garbage collector.
+	// It represents the GOGC environment variable. The default value is 100.
+	// Setting this to a negative value will effectively turn off the garbage collector unless the soft
+	// memory limit is reached.
+	// Please read the Go GC Guide for more info: https://tip.golang.org/doc/gc-guide
+	GarbageCollectorTriggerPercent int `mapstructure:"gc_percent"`
+}
```

Comment:
Why is it necessary to define configuration for these two fields when they are built into the Go runtime with the environment variables GOMEMLIMIT and GOGC?